src/vl-loader.js

```diff
@@ -0,0 +1,61 @@
+import { VlElement, define } from '/node_modules/vl-ui-core/dist/vl-core.js';
+
+/**
+ * VlLoader
+ * @class
+ * @classdesc De loader component is een animatie dat aangeeft dat een pagina of website aan het laden is.
+ * 
+ * @extends VlElement
+ * 
+ * @property {boolean} data-vl-light - Attribuut wordt gebruikt om een alternatieve rendering te bekomen voor op een te zorgen dat de radio getoond wordt als een block element en bijgevolg de breedte van de parent zal aannemen.
```

Comment:
```suggestion
 * @property {boolean} data-vl-light - Attribuut wordt gebruikt om een alternatieve rendering te bekomen. Hierdoor zal de radio getoond wordt als een block element en bijgevolg de breedte van de parent aannemen.
```
Beetje duidelijker?

Reply:
Ik denk dat die property ook iets zegt over de gebruikte kleuren (Alternative appearance of the loader with white elements on a dark background color.)

Reply:
Het laatste stuk van de tekst hoorde daar niet thuis. Ik heb het aangepast naar: `Attribuut wordt gebruikt om een alternatieve rendering te bekomen in combinatie met een donkere achtergrond`.